introduction.md

```diff
@@ -6,33 +6,44 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB **software** integrates different home automation systems, devices and technologies into a single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+## Getting Started
 
-## openHAB Structure
-
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+Now that you know what openHAB is, you may be wondering how to get started.
+We highly recommend that you read the next article titled [Concepts]({{base}}/concepts/index.html).
+This article introduces a number of important ideas that will help you as you install and begin to configure openHAB for the first time.
+openHAB runs on many popular platforms including Linux, Windows and Mac OSx.
+You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/intex.html) article.
+Many people find that the simplest way to experiment with openHAB and learn what it can do without making a significant investment is to get a [Raspberry Pi](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html), a "hastle-free openHAB setup".
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system capable of automating your entire home.
 
-openHAB is highly modular software, which means that the base installation (the "runtime") can be extended through different kinds of "add-ons", either to communicate with new home automation solution, or to offer a new kind of user interface.
+Once you have openHAB up and running, the [Configuration]([[base]]/configuration/index.html) tells you everything you need to know to get your openHAB installation talking to different [Things]([[base]]/configuration/things.html) around your home.
```

Comment:
> the chapter "Configuration"

or 

> the articles under the Configuration chapter

Reply:
Here and in the next sentence: Instead of things you should mention devices. openHAB talks to devices and aspects of devices are displayed and controlled via sitemaps. Somewhere I've already written a chapter on that.

Reply:
Regarding article, chapter, section...

To kick this off, you asked me to add a new chapter on "Getting Started", so I adopted that language in this article.  My preference would have been to call it a section.  So please help me understand the preferred terminology for the documentation.

If this was a book, the hierarchy would be as follows:
Book
  Chapter
    Section
      Sub-section
        Paragraph
          Sentence

OR, we could adopt
Documentation
  Article
    Section
      Sub-section
        Paragraph
         Sentence

OR something else?

WDYT?

Reply:
Regarding Things -> devices, this issue will be addressed in my next commit.

Reply:
@ThomDietrich, when you get time, please comment on my question above.  This is the last open item on this PR.

Reply:
Ah, to be honest I didn't think when using this word. I believe I was thinking of a section or even just a paragraph...

I'll remember to use the right term next time.

Reply:
Let's see if we can standardize on:
Documentation
Article
Section
Sub-section
Paragraph
Sentence

In this case, Article is the correct term and that is what I will use in my next commit.